### PR TITLE
Lesson 1 7

### PR DESCRIPTION
**Pull request by aligiselle**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,18 +2,53 @@ import React from 'react';
...
+      setTodoList(result.data.todoList);
+      setIsLoading(false);
+      });
+  }, []);
+
+  React.useEffect (() => {
+    if (!isLoading) {
+      localStorage.setItem('savedTodoList', JSON.stringify(todoList))
+    }
+  }, [todoList] );
```
Comment (jkylehumphrey) [inline src/App.js]: You need to add isLoading as dependency to this useEffect, so the result would be `[todoList,isLoading]`

**Review by jkylehumphrey — CHANGES_REQUESTED** [review]
I'm so sorry I've taken so long to get to these reviews.  Code looks good and I'll do better moving forward.

**Comment by jkylehumphrey** [comment]
Please make sure to merge the PR once you've made the changes.

File: src/App.js (hunk 1 above)
Comment (aligiselle) [reply src/App.js]: Thank you so much. No worries, I too have been having my fair share of challenges. I will make the necessary changes and merge.

**Pull request merged**